### DISCUSSION TITLE
[Feature Request] Collapsible rows in the Table Panel

### DISCUSSION THREAD

**Discussion by AbelThorne** [grafana/grafana#48967] [Dashboards, Panels & Visualizations]
Space can become a premium in a dashboard when you try to pack as much information but still keeping it readable. 

For instance, when a table panel has too many columns, users will have to scroll right and left to have a complete view which is not ideal. 
A good way to save space would be to allow collapsible rows in Table Panel : instead of having your panel stretch in length when the number of columns increase, maybe have a subset of columns displayed by default and then another subset that will be displayed when the user unfold the row to see more. This could be a kind of "wrap rows" option that would replace the scroll bar

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
